cocoapods/spec/dependabot/cocoapods/update_checker_spec.rb

```diff
@@ -77,13 +79,65 @@
   end
 
   let(:cocoapods_all_pods) do
-    fixture("cocoapods", "all_pods.txt")
+    fixture("cocoapods", "all_pods", "all_pods.txt")
   end
 
-  before do
+  let(:cocoapods_deprecated_specs) do
+    fixture("cocoapods", "specs", "deprecated_podspecs.txt")
+  end
+
+  let(:cocoapods_version_yaml) do
+    fixture("cocoapods", "CocoaPods-Version.yml")
+  end
+
+  def stub_cocoapods_cdn_requests
     all_pods_url = "https://cdn.cocoapods.org/all_pods.txt"
```

Comment:
NAB, but we should probably use the `#{COCOAPODS_CDN_HOST}` here too
```suggestion
    all_pods_url = "#{COCOAPODS_CDN_HOST}/all_pods.txt"
```